tests/analyze.R

```diff
@@ -106,20 +105,45 @@ ProcessAll = function(ctx) {
                   proportion = rappor$proportion,
                   dist = "rappor")
 
-  # Fill in zeros for values missing in RAPPOR.  It makes the ggplot bar plot
-  # look better.
+  # Fill in zeros for values missing in RAPPOR and in actual data
```

Comment:
I thought this was constructed so that rappor is a subset of actual.  "actual" comes from the true data I think, not the candidates?